src/v2/Components/ArtworkFilter/ArtworkFilters/ArtistNationalityFilter.tsx

```diff
@@ -8,11 +10,17 @@ export interface ArtistNationalityFilterProps {
 export const ArtistNationalityFilter: React.FC<ArtistNationalityFilterProps> = ({
   expanded,
 }) => {
+  const { selectedFiltersCounts } = useArtworkFilterContext()
+  const label = getFilterLabelWithCounts(
+    "Artist Nationality or Ethnicity",
+    selectedFiltersCounts.artistNationalities
+  )
+
```

Comment:
**thought**: can we place all this logic to the `Results Filter`?

Reply:
We can't move this logic to ResultsFilter because not all filters use `ResultsFilter` (some of them use `FilterExpandable`).
And I don't think this is a good idea to move this logic to `FilterExpandable` because it isn't related to filter context and it's used just to render expandable filters

Reply:
Got it. Thanks for the explanation